### ISSUE TITLE
Logistic Regression Inaccurate

### ISSUE THREAD

**Issue by sampepose** [sampepose/digit-recognizer#1]
My implementation of softmax logistic classifier is ~70% accurate, whereas the scikit-learn classifier is ~90% accurate. I'm not sure why this is happening. I can get to ~80% accuracy if I remove the regularization.

**Comment by dvfabbri** [comment]
Hmm, lets talk about this tomorrow
